tests/Unit/DependencyInjection/HttplugExtensionTest.php

```diff
@@ -449,4 +450,67 @@ public function testBatchClientCanBePublic()
             $this->assertFalse($this->container->getDefinition('httplug.client.acme.batch_client')->isPrivate());
         }
     }
+
+    /**
+     * @dataProvider provideVcrPluginConfig
+     * @group vcr-plugin
+     */
+    public function testVcrPluginConfiguration(array $config, array $services, array $arguments = [])
+    {
+        $prefix = 'httplug.client.acme.vcr';
+        $this->load(['clients' => ['acme' => ['plugins' => [['vcr' => $config]]]]]);
+        $this->assertContainerBuilderHasService('httplug.plugin.vcr.recorder.in_memory', InMemoryRecorder::class);
+
+        foreach ($services as $service) {
+            $this->assertContainerBuilderHasService("$prefix.$service");
```

Comment:
we do not use the inline string expansion thing. can you please rewrite this as `$prefix.'.'.$service`? (its a bit weird, but i'd like to keep a consistent code style)